bin/codecov

```diff
@@ -20,7 +20,8 @@ var args = argv.option([
   {name: 'slug', short: 'r', type: 'string', description: "Specify repository slug for Enterprise ex. owner/repo"},
   {name: 'url', short: 'u', type: 'string', description: "Your Codecov endpoint"},
   {name: 'flags', short: 'F', type: 'string', description: "Codecov Flags"},
-  {name: 'dump', type: 'boolean', description: "Dump collected data and do not send to Codecov"}
+  {name: 'dump', type: 'boolean', description: "Dump collected data and do not send to Codecov"},
+  {name: 'y', type: 'string', description: "Configuration file Used to specify the location of the .codecov.yml config file. Defaults to codecov.yml and .codecov.yml"}
```

Comment:
Could have name as `yml` and `short: 'y'`.

So will still remain consistent with the bash config but has a nicer name.

Forgot about this 😝